operator/v1/types_dns.go

```diff
@@ -376,6 +398,18 @@ const (
 	NetworkResolverType   UpstreamType = "Network"
 )
 
+// +kubebuilder:validation:Enum:=None;ForceTCP;""
```

Comment:
I don't think the Enum needs `""` if it it omitempty.

Reply:
The update looks fine in my opinion.